nilearn/plotting/img_plotting.py

```diff
@@ -1041,7 +1041,8 @@ def plot_connectome(adjacency_matrix, node_coords,
                     figure=None, axes=None, title=None,
                     annotate=True, black_bg=False,
                     alpha=0.7,
-                    edge_kwargs=None, node_kwargs=None):
+                    edge_kwargs=None, node_kwargs=None,
+                    colorbar=False):
```

Comment:
> There is no overlapping in the first pictures you posted. Did you change the size of the figure?

I tweaked a bit manually.

> This is not a great argument about which default to chose ...

I am open to any choice. Just that I mentioned what is happening